### PR TITLE
Maintainer Diversity Best Practices

### PR DESCRIPTION
**Pull request by tkuhrt**

### CODE REVIEW

**Review by petermetz — APPROVED** [review]
LGTM!

**Comment by arsulegai** [comment]
I would like to add "to consider timezone constraints" in choosing the best time to meet.

**Comment by tkuhrt** [comment]
> I would like to add "to consider timezone constraints" in choosing the best time to meet.

@arsulegai : Addressed via https://github.com/hyperledger/toc/pull/272/commits/c012a9416a268b52b563b5b3b0c945146f7dbc49

**Review by swcurran — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]
LGTM

File: guidelines/maintainer-diversity-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    - Webinars
+    - Meetups
+    - Documentation sprints
+    - Events
+    - Conferences
+- New Contributors
+    - Good first issues
+	    - Provide details and suggested directions for solution
+    - Hactoberfest
+    - Frequent users
```
Comment (arsulegai) [inline guidelines/maintainer-diversity-best-practices.md:94]: [nitpick] I remember one of the previous sessions with the AnitaB org enthusiasts.

**Review by arsulegai — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

**Review by VRamakrishna — APPROVED** [review]

**Review by samswag — COMMENTED** [review]
It looks good, but I recommend implementing "contributor office hours" as part of community building to help more contributors feel welcome and improve retention.

It will be good to spell it out.

File: guidelines/maintainer-diversity-best-practices.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    - Celebrate Milestones
+    - Badging
+- Distribute Responsibilities
+- Empower Contributors
+- Delegate Tasks
+    - Issue Manager
+    - FAQ Manager
+    - Writing Documentation
+    - Reviewing Pull Requests
+- Community Building
```
Comment (tkuhrt) [inline guidelines/maintainer-diversity-best-practices.md:30]:
```suggestion
- Community Building
    - Pair Programming Sessions
    - Contributor Office Hours
```

**Comment by tkuhrt** [comment]
> It looks good, but I recommend implementing "contributor office hours" as part of community building to help more contributors feel welcome and improve retention.
> 
> It will be good to spell it out.

Thanks, I added it.

**Pull request merged**